proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -2282,6 +2437,232 @@
       }
     }
   },
+  "components/separator": {
+    "utrecht": {
+      "separator": {
+        "color": {
+          "$type": "color",
+          "$value": "{purmerend.color.black-2}"
```

Comment:
Je stelt hier een brand token in, terwijl de voorkeur is is om common tokens te gebruiken. Was een common token niet mogelijk?

Als je een brand token gebruikt, dan is de kans groot dat het een contrastprobleem oplevert in dark mode.